Mathlib/Init.lean

```diff
@@ -26,6 +26,9 @@ public import Mathlib.Tactic.Linter.Style
 -- This import makes the `#min_imports` command available globally.
 public import Mathlib.Tactic.MinImports
 public import Mathlib.Tactic.TacticAnalysis.Declarations
+-- This is a redundant import, but it is needed so that
+-- the linter doesn't complain about `ParseCommand` not importing `Header`.
```

Comment:
```suggestion
-- the linter doesn't complain about `ParseCommand` not importing `Header`.
-- This can be removed after https://github.com/leanprover-community/mathlib4/pull/32419
```